airbyte-integrations/connectors/source-mongodb-v2/src/main/resources/spec.json

```diff
@@ -189,6 +189,19 @@
           "default": [],
           "order": 12,
           "group": "advanced"
+      },
+      "schema": {
+          "title": "Schema",
+          "description": "Hardcoded schema for each stream. If enabled, all streams must be annotated with a schema.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "minItems": 0,
+          "uniqueItems": true,
+          "default": [],
+          "order": 13,
+          "group": "advanced"
```

Comment:
Can this be an object instead of an array?  Airbyte's UI for array editing sucks.

Reply:
Or an array of objects or something?

Reply:
Yeah I'll look in to this